charts/kubewarden-controller/chart-values.yaml

```diff
@@ -87,26 +87,24 @@ resources:
     requests:
       cpu: 250m
       memory: 50Mi
-# experimental Kubewarden features. These features are not recommended for
-# production yet, and are not covered by SemVer guarantees.
-experimental:
-  auditScanner:
-    enable: false
-    # The default audit-scanner ServiceAccount is bound to the ClusterRoles:
-    # - view: Allows read-only access to most objects in a namespace.
-    #   Does not allow viewing secrets, roles or role bindings.
-    # - audit-scanner-cluster-role: Allows read-write to Kubewarden resources
-    #   and PolicyReports
-    serviceAccountName: audit-scanner
-    image:
-      # The registry is defined in the common.cattle.systemDefaultRegistry value
-      # kubectl image to be used in the pre-delete helm hook
-      repository: "kubewarden/audit-scanner"
-      tag: "latest"
-      pullPolicy: IfNotPresent
-    cronJob:
-      schedule: "*/60 * * * *" # every 60 minutes
-      failedJobsHistoryLimit: 5
-      successfulJobsHistoryLimit: 3
-    containerRestartPolicy: Never
-    skipAdditionalNamespaces: []
+
+auditScanner:
+  enable: false
+  # The default audit-scanner ServiceAccount is bound to the ClusterRoles:
+  # - view: Allows read-only access to most objects in a namespace.
+  #   Does not allow viewing secrets, roles or role bindings.
+  # - audit-scanner-cluster-role: Allows read-write to Kubewarden resources
+  #   and PolicyReports
+  serviceAccountName: audit-scanner
+  image:
+    # The registry is defined in the common.cattle.systemDefaultRegistry value
+    # kubectl image to be used in the pre-delete helm hook
+    repository: "kubewarden/audit-scanner"
+    tag: "latest"
+    pullPolicy: IfNotPresent
+  cronJob:
+    schedule: "*/60 * * * *" # every 60 minutes
+    failedJobsHistoryLimit: 5
+    successfulJobsHistoryLimit: 3
+  containerRestartPolicy: Never
+  skipAdditionalNamespaces: []
```

Comment:
this is the same list of ignored namespaces, the default list is in `global.skipNamespaces` in `values.yaml` (the one that gets shipped).

In kubewarden-controller we have `auditScanner.skipAdditionalNamespaces` (the chart specific one you are seeing here), and in kubewarden-defaults we have `recommendedPolicies.skipAdditionalNamespaces`.

documented them in https://github.com/kubewarden/helm-charts/pull/259/commits/00a2659a12ec0b7dafdcbf76eebf90679fee4295.

Reply:
Wait:

> Additional namespaces that the audit scanner will scan:
> skipAdditionalNamespaces: []

I find it confusing, because the variable has `skip` inside of its name. I was expecting 

> Additional namespaces that the audit scanner will **not** scan:

Reply:
whelp yes! fixed in https://github.com/kubewarden/helm-charts/pull/259/commits/5b82a397ca1b5cf4b6d663596f70ab6de9ff2015.